frontend/src/components/Search.module.scss

```diff
@@ -4,6 +4,10 @@
   height: 37px !important;
 }
 
-.wide {
+.smallwide {
   width: 260px !important;
 }
+
+.bigwide {
```

Comment:
:nitpick: also there are others that may fit the heading

```

.font-heading-3xs
.font-heading-2xs
.font-heading-xs
.font-heading-sm
.font-heading-md
.font-heading-lg
.font-heading-xl
.font-heading-2xl
.font-heading-3xl 

```

Reply:
Here we are trying to control the width of the search bar and not the text inside of it.

Reply:
ditto